### PR TITLE
2024 IONIQ 6

### PR DESCRIPTION
**Pull request by riley-williams**
**Car**

US 2024 IONIQ 6 (Limited AWD tested)

**Checklist**

- [x] added entry to CAR in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [ ] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [x] route with openpilot: 3ff5830d2216f742/00000001--bd301a1172
- [x] route with stock system: 3ff5830d2216f742/00000003--d8d4068cf8
- [x] car harness used: Hyundai P

**Notes**
- Minor tweaks to CarSpecs for IONIQ 6 reflect public info instead of cloning the subtly different IONIQ 5 specs
- Both model years have identical fingerprints, and the '24 is currently ID'd as a '23. The marketing materials for the two model years also seem to have no differences.

### CODE REVIEW

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -314,8 +314,10 @@ class CAR(Platforms):
     flags=HyundaiFlags.EV,
   )
   HYUNDAI_IONIQ_6 = HyundaiCanFDPlatformConfig(
-    [HyundaiCarDocs("Hyundai Ioniq 6 (with HDA II) 2023", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_p]))],
-    HYUNDAI_IONIQ_5.specs,
+    [HyundaiCarDocs("Hyundai Ioniq 6 (with HDA II) 2023-24", "Highway Driving Assist II", car_parts=CarParts.common([CarHarness.hyundai_p]))],
+    # Average of min/max mass of SEL/LTD models. Significantly lighter SE Standard Range should be uncommon
+    # https://www.hyundainews.com/assets/documents/original/53031-2023IONIQ6ProductSpecs093022.pdf
+    CarSpecs(mass=2004, wheelbase=2.95, steerRatio=14.27, tireStiffnessFactor=0.65),
```
Comment (riley-williams) [inline selfdrive/car/hyundai/values.py:320]: Expand up to see old values in IONIQ 5. Not sure if this is the right way to pick a weight, or what effect it has. The IONIQ 5 is heavier, but it looks like a value on the lower side was picked.

**Comment by sshane** [comment]
This good to go?

**Comment by sshane** [comment]
I don't see any 2024s in our data, but a lot of routes are missing the VIN due to some issue in the comma power (we should figure out how to get the VIN without that). Do you have a route of a 2024?

**Comment by riley-williams** [comment]
@sshane apologies for the slow progress getting these recorded. Added routes with and without openpilot. The without is unfortunately not very long, so I can get another if needed

**Comment by riley-williams** [comment]
Also not sure if you want this added to routes.py since there is already a route for the (identical?) 2023 model. Looking at the file, it seems like the list is being kept as short as possible.

**Comment by sshane** [comment]
Interesting that the 2024 uses the same FW from our first port: https://github.com/commaai/openpilot/pull/28915

LGTM:

| Dongle, platform, VIN                                                                                                      | Name from VIN 🆚 CarDocs   | Segments                                   | Bad seg tags                                                                                  | Good seg tags                                                                         |
|:---------------------------------------------------------------------------------------------------------------------------|:---------------------------|:-------------------------------------------|:----------------------------------------------------------------------------------------------|:--------------------------------------------------------------------------------------|
| [3ff5830d2216f742](https://useradmin.comma.ai/?onebox=3ff5830d2216f742)<br><sub>HYUNDAI_IONIQ_6<br>000000000........</sub> | None 🆚<br>None            | 43 routes,<br>968 segments<br>(16.1 hours) | - [spotty FW: 47](https://useradmin.comma.ai/?onebox=3ff5830d2216f742%7C00000000--d46df1ae91) | - valid torque params: 959<br>- all lat active: 501<br>- no input all lat active: 112 |

**Pull request merged**